tasks/list_queues.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+
+
+def list_queues(project,
+                location):
+    # [START cloud_tasks_list_queues]
+    """List all task queues."""
+
+    from google.cloud import tasks_v2
+
+    # Create a client.
+    client = tasks_v2.CloudTasksClient()
+
+    # Construct the fully qualified location path.
+    parent = client.location_path(project, location)
+
+    # Use the client to obtain the queues.
+    response = client.list_queues(parent)
+
+    # Convert the response to a list containing all queues.
+    queue_list = list(response)
+
+    # Print the results.
+    for queue in queue_list:
```

Comment:
Can we add a conditional for check if any queues were found and if not print 'No queues found!'